packages/@react-types/shared/src/collections.d.ts

```diff
@@ -141,3 +141,8 @@ interface AsyncListProps<T> {
 }
 
 declare function useAsyncList<T>(opts: AsyncListOptions<T>): AsyncListProps<T>;
+
+export interface CollectionsContextValue {
+  sectionComponent?: string | JSXElementConstructor<any>,
+  itemComponent?: string | JSXElementConstructor<any>
+}
```

Comment:
should props be optional with default? something like
itemComponent: ItemComponent = 'div'

Reply:
Eh... I think doing nothing by default is ok for now. When used inside a normal collection component we don't use the result of the rendering anyway.